recovery_backend/backend/configmap.go

```diff
@@ -69,7 +70,10 @@ func (rb *RecoveryBackend) CreateConfigMap(hostname, version string) error {
 		}
 	}
 
-	return nil
+	_, err = datastore.VerifyCreation(configMapName, "configmap", func(name string) (runtime.Object, error) {
+		return rb.Datastore.GetConfigMapRO(rb.Namespace, name)
+	})
+	return err
```

Comment:
Don't feel we need this. (I know in our current implementation in other places, all have this check; however, don't we just rely on the error returned by K8s API client?)

Reply:
cc @joshimoo @shuo-wu

Reply:
For creation, I think we don't need the `VerifyCreation`. But not sure if there is any context.

Reply:
We don't need this here. But as I mentioned, I am fine with having verification in the datastore Creation/Update functions.

Reply:
Agreed.